packages/flutter/lib/src/widgets/app.dart

```diff
@@ -1778,9 +1778,11 @@ class _WidgetsAppState extends State<WidgetsApp> with WidgetsBindingObserver {
           );
         },
       );
+    } else if (widget.title == null && kIsWeb) {
+      title = null;
```

Comment:
you should probably merge this `else if` case to the `if` case, setting this to null will drop the entire subtree

Reply:
Great catch!